### PR TITLE
docs: adds ngx-cookie-service library to resources list

### PR DESCRIPTION
**Pull request by pavankjadda**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information
Adding a Angular [ngx-cookie-service](https://github.com/stevermeister/ngx-cookie-service) library to the resources list. The library that can be used to read, set and delete browser cookies

### CODE REVIEW

File: aio/content/marketing/resources.json [hunk 1]
```diff
@@ -547,6 +547,11 @@
             "title": "NGX-YOUI",
             "url": "https://ngx-youi.github.io/NGX-YOUI/"
           }
```
Comment (jessicajaniuk) [inline aio/content/marketing/resources.json:549]:
```suggestion
          },
```

**Review by jessicajaniuk — CHANGES_REQUESTED** [review]
This needs a quick comma add, and the tests should be green.

**Comment by jessicajaniuk** [comment]
@pavankjadda Can you switch that second commit to a fixup? It should resolve the lint issues here and make it a clean single commit.

**Comment by pavankjadda** [comment]
> @pavankjadda Can you switch that second commit to a fixup? It should resolve the lint issues here and make it a clean single commit.

Sorry. You mean squash them into one commit and force push?

**Comment by jessicajaniuk** [comment]
> 

That would work, too. Whichever you prefer.

**Comment by jessicajaniuk** [comment]
@pavankjadda Looks like there's now an additional third commit here merging main into your branch. I can help with a rebase, if you'd like.

**Comment by pavankjadda** [comment]
> @pavankjadda Looks like there's now an additional third commit here merging main into your branch. I can help with a rebase, if you'd like.

Please do. I am trying to do it on local machine. It's taking forever to clone.

**Comment by jessicajaniuk** [comment]
@pavankjadda Just rebased for you. Should be all set now. I'll watch CI and then approve.

**Comment by pavankjadda** [comment]
> @pavankjadda Just rebased for you. Should be all set now. I'll watch CI and then approve.

Thanks :)

**Review by jessicajaniuk — APPROVED** [review]
LGTM :cookie:

**Review by mgechev — APPROVED** [review]

**Comment by jessicajaniuk** [comment]
This PR was merged into the repository by commit a4deac3fdab83f77324a9e76aec91799d8d6eecf.

**Pull request closed without merging**